content/en/container-orchestration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Container Orchestration
+status: Completed
+category: Concept
+---
+
+## What it is
+[Container](https://glossary.cncf.io/container/) orchestration automates container lifecycle management tasks such as provisioning, deployment, scaling, and networking in dynamic environments. 
+It functions like a data center operating system (OS), managing [containerized](https://glossary.cncf.io/containerization/) applications that run across a [distributed system](https://glossary.cncf.io/distributed-systems/) (just like the OS on your laptop that manages your apps).
+
+## Problem it addresses 
+Container usage began to grow significantly, bringing varying degrees of orchestration and cluster management capabilities. 
+There were tools that made it simple to run containers within virtual machines, but communicating all of this into the outside world and sharing secrets remained difficult. 
+A platform was required to run, manage, operate, and support the containers throughout their lifecycle.
+
+## How it helps
```

Comment:
If it is declarative or imperative does not really matter regarding how an orchestrator helps. I would more likely focus on the fact that the orchestrator takes care that all containerized apps are in a certain state. The fact that the orchestrator itself is self healing, does not necessarily ensure that the cluster remains in the desired state. Also cluster is a term that you used the first time in relation to container orchestration. This might confuse readers, and I am not entirely if all container orchestrators are clusters.

Reply:
Okay, will modify according to your suggestions

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Brittle